rts/Lua/LuaHandle.cpp

```diff
@@ -2428,6 +2428,27 @@ void CLuaHandle::ViewResize()
 	RunCallIn(L, cmdStr, 1, 0);
 }
 
+/*** Called whenever fonts are updated. Signals the game display lists
+ *   and other caches should be discarded.
+ *
+ * Gets called before other Update and Draw callins.
```

Comment:
Does this in practice mean "before any other callin", or can some callins happen inbetween?

Reply:
Its generally before other callins, I didn't want to give too much detail there, but it's called [here](https://github.com/beyond-all-reason/spring/blob/master/rts/Game/Game.cpp#L1454). Pretty much before everything else at Game::UpdateUnsynced, and I believe it's really before other Update and Draw callins.